virtwho/manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -284,10 +284,6 @@ def _is_rhsm_server_async(self, report, connection=None):
         else:
             self.logger.debug("Server does not have 'hypervisors_async' capability")
 
-        if is_async and os.environ.get('VIRTWHO_DISABLE_ASYNC', '').lower() in ['1', 'yes', 'true']:
-            self.logger.info("Async reports are supported but explicitly disabled")
-            is_async = False
-
```

Comment:
I'm OK with proposed change. What about writing some warning message to log file, that this env. variable is no longer supported?

Reply:
Makes sense to me!